paasta_tools/cli/cmds/status.py

```diff
@@ -838,6 +839,21 @@ def print_flink_status(
         output.append(f"    No other information available in non-running state")
         return 0
 
+    pod_running_count = pod_evicted_count = pod_other_count = 0
+    for pod in status.pod_status:
+        if pod["phase"] == "Running":
+            pod_running_count += 1
+        elif pod["phase"] == "Failed" and pod["reason"] == "Evicted":
+            pod_evicted_count += 1
+        else:
+            pod_other_count += 1
+    output.append(
+        "    Pods:"
+        f" {pod_running_count} running,"
+        f" {pod_evicted_count} evicted,"
```

Comment:
this one should probably be printed in the red color if the count of evicted pods is greater than 0?

Reply:
Yeah. I didn't add colors because `paasta status` for flink doesn't use any colors at all. But I guess it won't hurt to add some. Will update the PR in a moment!

Reply:
@gvolco paasta status for flink uses color when service is not running. I agree with @vkhromov using color will be a better idea.